### PR TITLE
Add changes to allow adding of Tile via user consent from app

### PR DESCRIPTION
**Pull request by alabiaga**
Add changes that allows the app to request that their tile be added if given user consent.

See https://developer.android.com/about/versions/13/features#quick-settings

image

image

### CODE REVIEW

**Comment by alabiaga** [comment]
cc\ @madeleinehoell

File: Quick-Settings/app/src/completed/java/com/google/android_quick_settings/QSDialogService.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 Google Inc.
+// Copyright 2022 Google Inc.
```
Comment (yaraki) [inline Quick-Settings/app/src/completed/java/com/google/android_quick_settings/QSDialogService.java:1]: We usually keep the year.

File: Quick-Settings/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -18,10 +18,16 @@
 <resources>
     <string name="app_name">Quick Settings Sample</string>
     <string name="tile_label">Quick Settings Sample</string>
-    <string name="app_greeting">To open the notification shade, swipe down from the top of the phone.
-        Expand the widget on the right side and then click <b>Edit</b>.
+    <string name="app_greeting"><b>Manual</b>\nOpen the notification shade to
```
Comment (yaraki) [inline Quick-Settings/app/src/main/res/values/strings.xml:21]: Does this `<b>` work without `Html.fromHtml`?

Also, the content should be surrounded with `<![CDATA[` ... `]]>` to use `<` in XMLs.

File: Quick-Settings/app/src/completed/java/com/google/android_quick_settings/QSDialogService.java (hunk 1 above)
Comment (alabiaga) [reply Quick-Settings/app/src/completed/java/com/google/android_quick_settings/QSDialogService.java:1]: reverted year changes. Thanks

**Review by yaraki — APPROVED** [review]

**Comment by alabiaga** [comment]
thanks @yaraki !

**Pull request merged**